web/src/locales/zh.json

```
@@ -109,6 +109,11 @@
       "create-a-member": "创建成员"
     }
   },
+  "amount-text": {
+    "memo": "MEMO",
+    "tag": "标签",
+    "day": "天"
```

English have a better visibility for there words than Chinese.
```suggestion
    "memo": "MEMO",
    "tag": "TAG",
    "day": "DAY"
```